### PR TITLE
Change the translated [KO] docs to be clearer.

### PR DESCRIPTION
**Pull request by MinByeongDon**
This PR change the translated [KO] docs to be clearer.

- Translate into more understandable sentences
- Remove unnecessary dart code files in the "KO" folder

Thanks

### CODE REVIEW

File: website/docs/essentials/testing/listen_provider.dart [hunk 1]
```diff
@@ -10,13 +10,13 @@ final provider = Provider((_) => 'Hello world');
 void main() {
   test('Some description', () {
     final container = createContainer();
-    /* SNIPPET START */
+/* SNIPPET START */
```
Comment (rrousselGit) [inline website/docs/essentials/testing/listen_provider.dart:13]: Could you revert those changes?
The macro relies on the indent to trim redundant whitespaces.

That's the difference between:

```dart
foo();
```

and
```dart
    foo();
```

Comment (rrousselGit) [reply website/docs/essentials/testing/listen_provider.dart:13]: Actually let me do that right now


```suggestion
    /* SNIPPET START */
```

File: website/docs/essentials/testing/listen_provider.dart [hunk 2]
```diff
@@ -10,13 +10,13 @@ final provider = Provider((_) => 'Hello world');
...
     container.listen<String>(
       provider,
       (previous, next) {
         print('The provider changed from $previous to $next');
       },
     );
-    /* SNIPPET END */
+/* SNIPPET END */
```
Comment (rrousselGit) [inline website/docs/essentials/testing/listen_provider.dart:20]:
```suggestion
    /* SNIPPET END */
```

File: website/docs/essentials/testing/widget_container_of.dart [hunk 3]
```diff
@@ -7,9 +7,9 @@ import 'widget_test.dart';
 
 void main() {
   testWidgets('Some description', (tester) async {
-    /* SNIPPET START */
+/* SNIPPET START */
```
Comment (rrousselGit) [inline website/docs/essentials/testing/widget_container_of.dart:10]:
```suggestion
    /* SNIPPET START */
```

File: website/docs/essentials/testing/widget_container_of.dart [hunk 4]
```diff
@@ -7,9 +7,9 @@ import 'widget_test.dart';
...
     final element = tester.element(find.byType(YourWidgetYouWantToTest));
     final container = ProviderScope.containerOf(element);
-    /* SNIPPET END */
+/* SNIPPET END */
```
Comment (rrousselGit) [inline website/docs/essentials/testing/widget_container_of.dart:13]:
```suggestion
    /* SNIPPET END */
```

**Comment by rrousselGit** [comment]
LGTM thanks!

File: website/docs/essentials/testing/listen_provider.dart (hunk 2 above)
Comment (MinByeongDon) [reply website/docs/essentials/testing/listen_provider.dart]: @rrousselGit 

Thank you for your review. :)

I wanted to remove the blank line at the bottom of the Before screen.
Think about whether it looks better like this.

Before:
image

After:
image

**Pull request merged**